docs/installation.rst

```diff
@@ -0,0 +1,38 @@
+Installation
+============
+
+There are a couple of ways to start using Pandas on Ray. Most users will want to
+install with ``pip``, but some users may want to build from the master branch on
+the `GitHub repo`_.
+
+Installing with pip
```

Comment:
Do we want to add documentation on the ray requirement here?

Reply:
Added, thanks!